.gitignore

```diff
@@ -1 +1,7 @@
 node_modules
+js
+ts/protocol
+ts/client/client.ts
```

Comment:
could this be `ts/**`

Reply:
No, there are some files in there that are not generated.

Reply:
I moved the generated files into separate folders so that we don't have to list individual files anymore.